<?xml version="1.0" encoding="UTF-8"?>

<launch>

    <include file="$(find flocking)/launch/leader.launch"/>
    <include file="$(find flocking)/launch/follower.launch">
        <arg name="model_to_be_followed_name" value="robot1" />
        <arg name="follower_model_name" value="robot2" />
    </include>
    <include file="$(find flocking)/launch/follower2.launch">
        <arg name="model_to_be_followed_name" value="robot1" />
        <arg name="follower_model_name" value="robot3" />
    </include>

</launch>
